cron_jobs/launch.ts

```diff
@@ -0,0 +1,33 @@
+import cronDemarchesSimplifiees from './cronDemarchesSimplifiees';
+import cronUniversityPayments from './cronUniversityPayments';
+import cronPsychologists from './cronPsychologists';
+
+const runJob = async (job) => {
+  await job();
+  process.exit(0);
+};
+
+if (process.argv.length < 3) {
+  console.log('Please specify the cron you want to launch:');
+  console.log('ts-node cron_jobs/launch.ts job_name');
```

Comment:
Are you sure it can be launched directly with `ts-node` if the package wasn't installed globally?
Shouldn't we use the path to ts-node inside the project?